registry/server/templates/routes/updateTemplate.ts

```diff
@@ -30,11 +30,22 @@ const validateRequestBeforeUpdateTemplate = validateRequestFactory([
 ]);
 
 const updateTemplate = async (req: Request<UpdateTemplateRequestParams>, res: Response): Promise<void> => {
-    await validateRequestBeforeUpdateTemplate(req, res);
+    try {
+        await validateRequestBeforeUpdateTemplate(req, res);
+    } catch(err) {
+        res.status(422).send(err);
+        return;
+    }
 
     const template = req.body;
     const templateName = req.params.name;
 
+    const countToUpdate = await db('templates').where({ name: templateName });
```

Comment:
I guess, this name of the variables which you wrote does not describe what inside of this variable
Because I guess here is `templates` well, they are entities, so `count` in this case does not describe what exactly we have there
I suggest to pay attention to the names of these variables in each of file